pennylane/interfaces/execution.py

```diff
@@ -39,6 +39,7 @@
 
 INTERFACE_MAP = {
     None: "Numpy",
+    "auto": "auto",

```

Comment:
What happens if `qml.execute` is called directly without using a QNode? Good to add tests for that case too.

Also, should `"auto"` be our default going forward?

Reply:
Added "auto" interface for all interface checks.

Yes, `"auto"` will be our default going forward.